### ISSUE TITLE
Maybe look at bringing in caret for ensemble modeling

### ISSUE THREAD

**Issue by danlwarren** [danlwarren/ENMTools#118]
Nice little tutorial here:

http://amunategui.github.io/blending-models/

It seems like we could bring in caret in such a way that you pass in a list of model types (with a default list, presumably).  This would likely require us to write a predict function for caret models onto raster models, but that shouldn't be too hard.  Depending on the predict syntax for caret, we might even be able to use one of the existing raster::predict functions by tricking it with a temp copy of the caret model with the class values reset.

Another nice thing about doing this is that it wouldn't necessarily just be for ensembles; we could use this to open up a whole bunch of other methods that caret is a wrapper for, just by passing a single model type in the list.

**Comment by danlwarren** [comment]
Russell found this, which seems potentially useful:

https://www.tidymodels.org/learn/

**Comment by danlwarren** [comment]
Even better, here's a whole book!

https://www.tmwr.org/

**Comment by rdinnager** [comment]
Yes! I just saw this pop up on my twitter feed as well. Got some reading to do!
